tests/testthat/test-geom-vline-hline-allow-none-1947.R

```diff
@@ -0,0 +1,77 @@
+
+test_that("geom_vline/geom_hline does not throw an error with ggplotly when no lines are found", {
```

Comment:
It'd be great if these tests:

- Were added to `testthat/test-ggplot-hline.R`
- Were more minimal (i.e., no need to use datatable or gapminder... please use minimal datasets like the other tests in `testthat/test-ggplot-hline.R`

Reply:
Thanks @cpsievert I'll work on these updates later today :)

Reply:
I am testing both hline and vline; I noticed there is a test-ggplot-vline.R file.

I simplified the data and corrected the expected results in the test assertion.

For now I've moved the test to the hline file as you suggested - I am expecting you to suggest separating these two into their respective files.

Reply:
Hi @cpsievert you want me to make any more updates on this pull request? 

I separated the tests into their respective files; vertical/horizontal.